### ISSUE TITLE
Mastodon: Redrafted posts aren't deleted

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#7940]
Mastodon provides a feature to "redraft" a post, which is basically a "delete + post" feature. Friendica receives the new post correctly, but fails to delete the previous one. Are we receiving a delete action for it?

**Comment by annando** [comment]
In the past it worked.

**Comment by MrPetovan** [comment]
Did anyone interact with the redrafted post?

**Comment by MrPetovan** [comment]
See example here: https://friendica.mrpetovan.com/display/64c00ddafe51e9e581a43242b13f9b6dfe7669ee

**Comment by MrPetovan** [comment]
Another example: https://friendica.mrpetovan.com/display/b9f5c781d8f4057ce167c7c268b1a852cae7c083

This issue seems to be specific to replies.

**Comment by annando** [comment]
It still works here, I just tested with a comment.

**Comment by MrPetovan** [comment]
Humph.

**Comment by annando** [comment]
Are you following these accounts? Possibly it's not us not performing the delete, but the other side not sending it to us?

**Comment by MrPetovan** [comment]
I am following neither of these accounts. Maybe this is why they don't send delete messages.

**Comment by annando** [comment]
I guess so - which in my mind is a bug. When I delete a comment on a post, then I have to send the delete message not only to my followers, but to the people in the post as well. (memo to myself: testing how we are doing it via AP)

**Comment by MrPetovan** [comment]
Should I open an issue at Mastodon?

**Comment by annando** [comment]
I guess you should.

**Comment by MrPetovan** [comment]
Mastodon issue: https://github.com/tootsuite/mastodon/issues/12659

**Comment by annando** [comment]
I don't think that we can do anything about this. So I guess you left this issue open as a reminder looking at the Mastodon issue?

**Comment by MrPetovan** [comment]
Absolutely. I understand that we correctly process deletion notices, but if we don't receive one, there;s nothing we can do about it.

**Comment by MrPetovan** [comment]
Removing from milestone since it depends on https://github.com/tootsuite/mastodon/issues/12659
